### DISCUSSION TITLE
Where can i find the logs of failed login attemps?

### DISCUSSION THREAD

**Discussion by tsator** [apache/superset#29379] [Q&A / Help]
Hi!, i need help for this.
I need to obtain the logs of failed authentication attemps , to determine the cause of the failure (Incorrect username, incorrect password, etc)

<img width="208" alt="image" src="https://github.com/apache/superset/assets/135351271/d224cc76-7c3c-4b21-823c-83d6709e819a">

<img width="442" alt="image" src="https://github.com/apache/superset/assets/135351271/ae7ddee0-4390-41f3-86e5-14f296168af4">

I already looked at the logs table (bi_fed_metadata.logs) but it only returns logs of the actions the user takes once they have successfully logged in

**Comment by andreyks95** [comment]
The same question
